client/my-sites/plans/jetpack-plans/product-store/style.scss

```diff
@@ -5,6 +5,25 @@
 	display: flex;
 	flex-direction: column;
 
+	.header {
+		margin-bottom: 36px;
+	}
+
+	.header .formatted-header {
+		margin: 0;
+	}
+
+	.header .formatted-header__title {
+		margin: 0 auto !important;
+		font-size: $font-title-large !important;
+		line-height: rem( 38px ) !important;
+
+		@include break-large {
+			font-size: $font-headline-small !important;
+			line-height: rem( 40px ) !important;
```

Comment:
Is there a way to avoid `!important` by using some [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity)?

Reply:
@manzoorwanijk thanks for the suggestion. I changed the selector to be more specific so I don't have to use the !important. 👍

Reply:
Great. Thank you.